examples/react-app/src/Avatars.tsx

```diff
@@ -0,0 +1,28 @@
+import { PlmgAvatar } from '@ducky/plumage-react';
+
+const styles = {
+  marginLeft: '-15px',
+};
+
+export default function Avatars() {
+  return (
+    <>
+      <h3>Avatars</h3>
+      <div style={{ display: 'flex' }}>
+        <PlmgAvatar
+          size={'medium'}
+          imageUrl="https://static.ducky.eco/icons/maskable_icon_192.png"
+        ></PlmgAvatar>
+        <PlmgAvatar style={styles} size={'medium'}></PlmgAvatar>
+        <PlmgAvatar style={styles} size={'medium'} userDeleted></PlmgAvatar>
+        <PlmgAvatar
+          size={'medium'}
+          style={styles}
+          imageUrl="https://static.ducky.eco/icons/maskable_icon_192.png"
+        ></PlmgAvatar>
+        <PlmgAvatar style={styles} size={'medium'}></PlmgAvatar>
+        <PlmgAvatar size={'medium'} style={styles} userDeleted></PlmgAvatar>
```

Comment:
Imagine the use case where these avatars would be grouped. It is unlikely we will define each one rather will have a list.
I suggest making a profiles object which contains information on users. Then have an array of avatars you'd like to show. Map that array drawing information from your profiles object.

Reply:
I've implemented your suggestion using a simple object (I considered calling a test API of users to get a more realistic complicated json object - but thought that might be too much clutter for the example)